etc/mpv.desktop

```diff
@@ -1,43 +1,314 @@
+Comment[yo]=Mu awọn fiimu ati awọn orin ṣiṣẹ
 Comment[zh_CN]=播放电影和歌曲
 Comment[zh_TW]=播放電影和歌曲
-Icon=mpv
-TryExec=mpv
-Exec=mpv --player-operation-mode=pseudo-gui -- %U
-Terminal=false
-Categories=AudioVideo;Audio;Video;Player;TV;
-MimeType=application/ogg;application/x-ogg;application/mxf;application/sdp;application/smil;application/x-smil;application/streamingmedia;application/x-streamingmedia;application/vnd.rn-realmedia;application/vnd.rn-realmedia-vbr;audio/aac;audio/x-aac;audio/vnd.dolby.heaac.1;audio/vnd.dolby.heaac.2;audio/aiff;audio/x-aiff;audio/m4a;audio/x-m4a;application/x-extension-m4a;audio/mp1;audio/x-mp1;audio/mp2;audio/x-mp2;audio/mp3;audio/x-mp3;audio/mpeg;audio/mpeg2;audio/mpeg3;audio/mpegurl;audio/x-mpegurl;audio/mpg;audio/x-mpg;audio/rn-mpeg;audio/musepack;audio/x-musepack;audio/ogg;audio/scpls;audio/x-scpls;audio/vnd.rn-realaudio;audio/wav;audio/x-pn-wav;audio/x-pn-windows-pcm;audio/x-realaudio;audio/x-pn-realaudio;audio/x-ms-wma;audio/x-pls;audio/x-wav;video/mpeg;video/x-mpeg2;video/x-mpeg3;video/mp4v-es;video/x-m4v;video/mp4;application/x-extension-mp4;video/divx;video/vnd.divx;video/msvideo;video/x-msvideo;video/ogg;video/quicktime;video/vnd.rn-realvideo;video/x-ms-afs;video/x-ms-asf;audio/x-ms-asf;application/vnd.ms-asf;video/x-ms-wmv;video/x-ms-wmx;video/x-ms-wvxvideo;video/x-avi;video/avi;video/x-flic;video/fli;video/x-flc;video/flv;video/x-flv;video/x-theora;video/x-theora+ogg;video/x-matroska;video/mkv;audio/x-matroska;application/x-matroska;video/webm;audio/webm;audio/vorbis;audio/x-vorbis;audio/x-vorbis+ogg;video/x-ogm;video/x-ogm+ogg;application/x-ogm;application/x-ogm-audio;application/x-ogm-video;application/x-shorten;audio/x-shorten;audio/x-ape;audio/x-wavpack;audio/x-tta;audio/AMR;audio/ac3;audio/eac3;audio/amr-wb;video/mp2t;audio/flac;audio/mp4;application/x-mpegurl;video/vnd.mpegurl;application/vnd.apple.mpegurl;audio/x-pn-au;video/3gp;video/3gpp;video/3gpp2;audio/3gpp;audio/3gpp2;video/dv;audio/dv;audio/opus;audio/vnd.dts;audio/vnd.dts.hd;audio/x-adpcm;application/x-cue;audio/m3u;
-X-KDE-Protocols=ftp,http,https,mms,rtmp,rtsp,sftp,smb,srt,rist
-StartupWMClass=mpv
+
+# Translators: Search terms to find this application. Don't translate the semicolons! The list MUST also end with a semicolon!
+Keywords=mpv;media;player;video;movie;audio;songs;music;tv;
+Keywords[pt_BR]=mpv;mídia;reproduzir;tocar;vídeo;filme;áudio;som;música;tv;
+
+# Dear volunteer translators, please add your language abbreviation and name here in alphabetical order.
+#
+# ca:
+# cs:
+# da:
+# de: Robin.antiX
+# fr: 
+# fr_BE: Wallon
+# it: Spartak77, Daddylibre, Madibi
+# da: troc
+# hi: ashish.kujur2
+# hu: savarall
+# ja:
+# or: ashish.kujur2
+# pl:
+# pt_BR: marcelocripe
+# pt: PPC/Zeh
+# ru: 
+# so: cismat10
+# sv: bittin
+# tr:
+# uk: laslo_one
+#
+# zh_CN:
+# zh_TW:
+# Note: Other languages have been translated with internet translators and need to be proofread or corrected by native speakers of the respective languages.
```

Comment:
This is not okay. Translations either need to be correct (done by a human) or we can't accept them.